internal/api/offlinecheck/helm.go

```diff
@@ -72,7 +92,7 @@ func CompareVersion(ver1, ver2 string) bool {
 	return false
 }
 
-func IsReleaseCertified(helm *release.Release, ourKubeVersion string) bool {
+func (checker OfflineChecker) IsReleaseCertified(helm *release.Release, ourKubeVersion string) bool {
```

Comment:
Same here `checker` isn't used.

Reply:
yep, there's no need for checker in offline mode. To use it, I need to remove registry package and replace it with this one